cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -47,20 +47,28 @@
 
    ========================================================================== #}
 
-{% macro _nav_list( nav_depth, nav_groups, language='en' ) %}
+{% macro _nav_list( nav_depth, nav_groups, nav_title='', language='en' ) %}
     {% for nav_group in nav_groups %}
     {% set group = nav_group.value %}
     <div class="{{ _classes( nav_depth, '-list' ) }}">
 
         {% if group.group_title %}
+        {% set aria_group_title = group.group_title ~ ' continued' if group.hide_group_title else group.group_title %}
         <h4 class="h5
                   {{ base_class ~ '_group-heading' }}
                   {{ base_class ~ '_group-heading' ~ '__hidden' if group.hide_group_title else ''}}"
-            {% if group.hide_group_title %} aria-hidden="true" {% endif %}>
+            id="{{ aria_group_title | slugify ~ '-menu' }}"
+            {% if group.hide_group_title %} aria-hidden="true" {% endif %}
+            aria-label="{{ aria_group_title | safe }}">
             {{ group.group_title | safe }}
         </h4>
         {% endif %}
-        <ul>
+        <ul
+            {% if aria_group_title %}
+            aria-labelledby="{{ aria_group_title | slugify ~ '-menu' }}"
+            {% else %}
+            aria-label="{{ nav_title | safe }}"
```

Comment:
@anselmbradford I changed it so that if there's no link group title whatsoever it falls back to the section's nav title (e.g. Practitioner Resources).